pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -286,11 +287,6 @@ public void setOffloadPolicies(@Suspended final AsyncResponse asyncResponse,
                                                             OffloadPolicies offloadPolicies) {
         validateTopicName(tenant, namespace, encodedTopic);
         validateAdminAccessForTenant(tenant);
-        validatePoliciesReadOnlyAccess();
-        checkTopicLevelPolicyEnable();
-        if (topicName.isGlobal()) {
-            validateGlobalNamespaceOwnership(namespaceName);
-        }
         internalSetOffloadPolicies(offloadPolicies).whenComplete((res, ex) -> {
```

Comment:
A `preValidation()` seems to be missing here

Reply:
Looks like the original PR also missed this
https://github.com/apache/pulsar/pull/9767/files#diff-f37aee2b7a1b61a339394b5ff47d7bd53133b58ec4c6faabea5375f5294d6048L311

Reply:
Please help add...